src/content/docs/fr/core-concepts/project-structure.mdx

```diff
@@ -57,41 +58,49 @@ Le code source de votre project se trouve dans le dossier `src/`. Il comprend en
 - [Des fichiers de style (CSS, Sass)](/fr/guides/styling/)
 - [Du Markdown](/fr/guides/markdown-content/)
 
-Astro traite, optimise et regroupe les fichiers `src/` pour créer le site web final qui est délivré au navigateur. Contrairement au répertoire statique `public/`, les fichiers `src/` sont traités et assemblés pour vous par Astro.
+Astro traite, optimise et regroupe vos fichiers `src/` pour créer le site web final qui est livré au navigateur. Contrairement au répertoire statique `public/`, les fichiers `src/` sont traités et assemblés pour vous par Astro.
 
-Quelques fichiers (comme les composants Astro) ne sont pas envoyés au navigateur tel quel, mais plutôt en temps qu'HTML statique. D'autres fichiers (CSS par exemple) sont envoyés au navigateur, mais peuvent être optimisés ou regroupés avec d'autres fichiers CSS pour améliorer les performances.
+Certains fichiers (comme les composants Astro) ne sont pas envoyés au navigateur tels qu'ils sont écrits, mais sont rendus en HTML statique. D'autres fichiers (CSS par exemple) sont envoyés au navigateur, mais peuvent être optimisés ou regroupés avec d'autres fichiers CSS pour améliorer les performances.
+
+:::tip
+Bien que ce guide décrive quelques conventions populaires utilisées dans la communauté Astro, les seuls répertoires réservés par Astro sont `src/pages/` et `src/content/`. Vous êtes libre de renommer et de réorganiser les autres répertoires de la manière qui vous convient le mieux.
+:::
 
 ### `src/components`
 
-**Les composants** sont généralement du code réutilisable pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants Frontend](/fr/core-concepts/framework-components/) comme React ou Vue. Il est commun de grouper et d'organiser tous les composants de votre projet dans ce dossier.
+**Les composants** sont, généralement, du code réutilisable pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants Frontend](/fr/core-concepts/framework-components/) comme React ou Vue. Il est courant de regrouper et d'organiser tous les composants de votre projet dans ce dossier.
```

Comment:
I don't understand why you use `composants Frontend`, because it's not really the translation of `UI framework components`. I'd translate it as `composants d'interface utilisateur de framework` (we could remove the `de framework` because it's not really pretty).
And you did a little mistake its `commun` instead of `courant`.
```suggestion
**Les composants** sont, généralement, du code réutilisable pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants d'interface utilisateur de framework](/fr/core-concepts/framework-components/) comme React ou Vue. Il est commun de regrouper et d'organiser tous les composants de votre projet dans ce dossier.
```

Reply:
I've gone back to the translation that was done before.